lib/Config/Menu.php

```diff
@@ -605,6 +610,7 @@ function newsletters() {
     });
     $data['segments'] = $segments;
     $data['settings'] = $this->settings->getAll();
+    $data['user_flags'] = $this->user_flags->getAll();
```

Comment:
Do we really need all flags in the template? Would it be better to explicitly pass only those we really need?